packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -606,14 +700,30 @@ export const BaseInput = React.forwardRef<HTMLInputElement, BaseInputProps>(
       hasPopup,
       popupId,
       isPopupExpanded,
+      tagRows,
       shouldIgnoreBlurAnimation,
+      setShouldIgnoreBlurAnimation,
       autoCapitalize,
       testID,
       ...styledProps
     },
     ref,
   ) => {
     const { theme } = useTheme();
+    const inputWrapperRef = React.useRef<HTMLDivElement>(null);
```

Comment:
HTMLDivElement seems like it could fail on native

Reply:
Changed it here. There's still some div references in Dropdown component for refs that are used on web only. I'll later refactor them out separately. I anyway want to take a task to move as much logic to floating ui as possible so those refs might be removed.

#1353